llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -3917,9 +3929,10 @@ void LoopVectorizationCostModel::collectLoopUniforms(ElementCount VF) {
                         << *I << "\n");
       return;
     }
-    if (isScalarWithPredication(I, VF)) {
-      LLVM_DEBUG(dbgs() << "LV: Found not uniform being ScalarWithPredication: "
-                        << *I << "\n");
+    if (isPredicatedInst(I, VF, true)) {
```

Comment:
```suggestion
    if (isPredicatedInst(I, VF, /*IsKnownUniform=*/true)) {
```

Reply:
Code is gone in the latest version, thanks!